web-frontend/src/pages/SelectAccount/SelectAccount.tsx

```diff
@@ -41,10 +41,15 @@ export default function SelectAccount() {
                 to={routeFor('home')}
                 state={{ nickname: account.nickname }}
               >
-                <div className="mb-4" key={account.nickname}>
+                <div className="mb-4 theme-light" key={account.nickname}>
                   <AccountSelector
-                    avatar={<FiUser className="text-neutral h-6 w-6" />}
-                    icon={<MassaToken size={24} />}
+                    avatar={<FiUser className="text-primary h-6 w-6" />}
+                    icon={
+                      <MassaToken
+                        size={24}
+                        className="bg-neutral rounded-full"
```

Comment:
![image](https://github.com/massalabs/thyra-plugin-wallet/assets/11590067/be2439a4-5883-404a-af29-c335ebe66001)
 we need, otherwise the icon is square and the color is bad.

Reply:
Ouch! Another issue in the ui-kit! We must fix there first.

I'll fix and keep you posted once again!

Reply:
yeah, in the story we add rounded-full, we should probably remove this.

Reply:
I have created issue https://github.com/massalabs/ui-kit/issues/145

Reply:
Closing here since we clarified in the call!